TheSyndicate/Program.cs

```diff
@@ -1,12 +1,33 @@
 ﻿using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+
 
 namespace TheSyndicate
 {
     class Program
     {
+        [DllImport("kernel32.dll", ExactSpelling = true)]
```

Comment:
I wonder if we can export this to another class?

Reply:
That sounds good.